### ISSUE TITLE
Does map editing corrupt the game or cause problems?

### ISSUE THREAD

**Issue by Whoeza** [dexyfex/CodeWalker#188]
Hi desyfex,

I watched your youtube tutorial. When you make a project and export a map after editing it in CodeWalker, is it safe to use in FiveM? Can I make a resource that uses the edited map?

**Comment by WEEZOOKARP** [comment]
yes its safe, the only things that really may be troublesome is if your texture dictionaries are too big to stream reliably but that shouldn't be a problem unless you start using custom textures or models
tl;dr your fine just chuck the ymap file into a resource and setup the manifest
